### PR TITLE
Closes AgileVentures/MetPlus_tracker#290

### PR DESCRIPTION
**Pull request by joaopapereira**

### CODE REVIEW

File: features/agency_admin_autorization.feature [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+      | Software Engineer - RoR   | Develop website using Ruby on Rails |
+
+    Given I am on the home page
+
+  @allow-rescue
+  Scenario:
+    Given I login as "jane@metplus.org" with password "qwerty123"
+    Then I should see "Signed in successfully."
+    And I go to the agency 'MetPlus' edit page
+    And I should see "You are not authorized to edit MetPlus agency."
```
Comment (patmbolger) [inline features/agency_admin_autorization.feature]: Misspelled 'authorization' in the file name.

**Comment by patmbolger** [comment]
Reviewed PR with one comment.

Also, a question: should we add tests to tasks_controller_spec.rb for each new task added to business_logic.rb?

**Comment by joaopapereira** [comment]
We already have tests for that in business logic module right?

**Comment by patmbolger** [comment]
We do have tests for all tasks in business_logic.rb.  I guess we should continue that practice.

**Comment by joaopapereira** [comment]
Going to merge PR

**Pull request merged**